examples/trivial/app/app.kt

```diff
@@ -8,21 +8,18 @@ import graphql.GraphQL
 data class Foo(val name: String, val age: Int)
 
 class Query {
```

Comment:
@hanneskaeufler Did you mean to pull in these changes?

Reply:
Yes, file needed to be reformatted to pass the linter